Updating the icons for deactivating/reactivating/managing users and in Help center

ahmedgulabkhan opened the pull request:
<!-- Describe your pull request here.-->
We're switching to new icons for deactivating/reactivating/managing users and in Help center.

The new icons to be used can be found [here](https://chat.zulip.org/#narrow/channel/101-design/topic/deactivate.20group.20icon/near/2025552)

### Icon changes:

In Organization Settings:
- Replaced `fa-user-times` with `user-x.svg` in both user and bot management panels
- Replaced `fa-user-plus` with `user-plus.svg` in both user and bot management panels
- Replaced `zulip-icon-user-cog` with `user-manage.svg`, and kept the current icon name
- Replaced `fa-pencil` with `user-manage.svg`(i.e., `zulip-icon-user-cog`) in both the user and bot management panels

In Help Center:
- Replaced `fa-user-times` with `user-x.svg` for deactivating/reactivating user, bot
- Replaced `fa-user-plus` with `user-plus.svg` for deactivating/reactivating user, bot
- Replaced `fa-pencil` with `user-manage.svg`(i.e., `zulip-icon-user-cog`) for managing user, bot


Fixes: <!-- Issue link, or clear description.--> Part of [#32887](https://github.com/zulip/zulip/issues/32887)

<!-- If the PR makes UI changes, always include one or more still screenshots to demonstrate your changes. If it seems helpful, add a screen capture of the new functionality as well.

Tooling tips: https://zulip.readthedocs.io/en/latest/tutorials/screenshot-and-gif-software.html
-->

**Screenshots and screen captures:**

| Before | After |
|----------|----------|
| ![1](https://github.com/user-attachments/assets/845cb8cf-ba44-44bd-a321-fa04096f2f78) | ![21](https://github.com/user-attachments/assets/ecdafe34-d397-4d22-8840-8554b419c00b) |
| ![2](https://github.com/user-attachments/assets/09ba7f81-5b71-49e9-a25a-841b6ab4f594) | ![22](https://github.com/user-attachments/assets/d890aed8-cd2b-4062-af15-9cbdf14a5935) |
| ![5](https://github.com/user-attachments/assets/07246ba0-3b2d-43ed-8ddb-6ca44a7ef96a) | ![23](https://github.com/user-attachments/assets/d608a810-3e5e-4a3c-a630-73c43226d5ff) |
| ![7](https://github.com/user-attachments/assets/ea8eb76b-b284-47ba-9895-50db94f6795f) | ![24](https://github.com/user-attachments/assets/98ae7d59-f18a-4f61-96d7-0dbe20f27c4f) |
| ![14](https://github.com/user-attachments/assets/c6fd544b-6b0b-4b98-a844-1f248a52bc8c) | ![25](https://github.com/user-attachments/assets/b9a203f8-9b89-46f1-8166-4d3a6db5e004) |
| ![Screenshot 2025-01-20 at 9 33 28 PM](https://github.com/user-attachments/assets/25824a52-cc00-42c7-9c65-29d54e7f76ab) | ![Invitations-Dark-After](https://github.com/user-attachments/assets/dd20b8b9-964b-40a1-836a-bf1d25202c4f) |
| ![Bots-light-before](https://github.com/user-attachments/assets/db27e220-8450-483e-83fb-dca2dac8bec8) | ![26](https://github.com/user-attachments/assets/2b7c128b-103a-4ccf-889e-0093ad8e5d12) |
| ![Bots-light-before-2](https://github.com/user-attachments/assets/af894a76-8bfa-418f-892b-99050d213af3) | ![27](https://github.com/user-attachments/assets/634b5604-8b94-457c-827d-67784e15e0d7) |
| ![Invitations-light-before](https://github.com/user-attachments/assets/2d408b2a-0db6-4e71-b8c9-9cc320e264c0) | ![Invitations-light-after](https://github.com/user-attachments/assets/ed769153-992d-487e-8450-54796341518d) |
| ![Deactivated-light-before](https://github.com/user-attachments/assets/c775ba49-5156-48e9-98a9-007346af1785) | ![28](https://github.com/user-attachments/assets/dec05b0e-8758-4a44-ad66-3b7f4f1483ce) |
| ![User-light-before](https://github.com/user-attachments/assets/66da4e68-8644-45a7-9ae3-b8b12ca4bed5) | ![29](https://github.com/user-attachments/assets/c64253cd-da42-470a-8170-2c3f3ec2d7bb) |
| ![Manage-user-before](https://github.com/user-attachments/assets/0c8ffe6f-ebcd-40f6-b7ae-f9bf566eefe9) | ![Manage-user-after](https://github.com/user-attachments/assets/aa7772e5-6e85-4bf7-97e3-3b8db22c22e0) |
| ![Edit-bot-before](https://github.com/user-attachments/assets/361904ae-1c9d-430b-96ab-0096fe1f5fa2) | ![Edit-bot-after](https://github.com/user-attachments/assets/0d63f1cb-64d7-49c1-8010-28e0277252cc) |
| ![Reactivate-bot-before](https://github.com/user-attachments/assets/085c09ec-08e2-4dc3-8957-28c548d0610f) | ![Reactivate-bot-after](https://github.com/user-attachments/assets/f892f983-373c-4598-bcfa-6ca01696fbd4) |
| ![Deactivate-bot-before](https://github.com/user-attachments/assets/49d8fc8d-06ba-4de4-95c0-7cfca502e8ac) | ![Deactivate-bot-after](https://github.com/user-attachments/assets/50083f6d-e71e-4c35-ad56-4e62502ece8a) |
| ![Deactivate-bot-before-2](https://github.com/user-attachments/assets/e0326864-6873-4ec9-82fd-ad8ce398ed1f) | ![Deactivate-bot-after-2](https://github.com/user-attachments/assets/9b3b08a8-c776-4008-84bb-5c7f36c4e8d5) |
| ![Deactivate-user-before](https://github.com/user-attachments/assets/fc7095d3-1fe6-491b-8148-e5f4fd488909) | ![Deactivate-user-after](https://github.com/user-attachments/assets/458041b9-c1be-40ec-a867-c688206bd770) |


The already existing icons in the 'Settings/Bots' page were not aligned vertically, although it was not clearly visible while using the old icons. And when I replaced the old icons with the new ones, I was able to see the difference in the alignment of the other icons present in this page (i.e; buttons with the class names `download_bot_zuliprc`, `copy_zuliprc`, `open_bots_subscribed_streams`). I have made all these icons to be aligned vertically. Below is a screenshot of the before/after of how they look right after adding the new icons, and after fixing the vertical alignment issue

| Before | After |
|----------|----------|
| ![Screenshot 2025-02-05 at 1 30 51 PM](https://github.com/user-attachments/assets/3b63bf9d-0646-444d-8951-37f47d846fbe) | ![Screenshot 2025-02-05 at 1 33 09 PM](https://github.com/user-attachments/assets/86783eda-6a51-40d1-bbac-bb47c70b2c1d) |

<details>
<summary>Self-review checklist</summary>

<!-- Prior to submitting a PR, follow our step-by-step guide to review your own code:
https://zulip.readthedocs.io/en/latest/contributing/code-reviewing.html#how-to-review-code -->

<!-- Once you create the PR, check off all the steps below that you have completed.
If any of these steps are not relevant or you have not completed, leave them unchecked.-->

- [x] [Self-reviewed](https://zulip.readthedocs.io/en/latest/contributing/code-reviewing.html#how-to-review-code) the changes for clarity and maintainability
      (variable names, code reuse, readability, etc.).

Communicate decisions, questions, and potential concerns.

- [x] Explains differences from previous plans (e.g., issue description).
- [ ] Highlights technical choices and bugs encountered.
- [ ] Calls out remaining decisions and concerns.
- [x] Automated tests verify logic where appropriate.

Individual commits are ready for review (see [commit discipline](https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html)).

- [x] Each commit is a coherent idea.
- [x] Commit message(s) explain reasoning and motivation for changes.

Completed manual review and testing of the following:

- [x] Visual appearance of the changes.
- [x] Responsiveness and internationalization.
- [x] Strings and tooltips.
- [x] End-to-end functionality of buttons, interactions and flows.
- [x] Corner cases, error conditions, and easily imagined bugs.
</details>

alya commented:
The icons don't look properly aligned within the buttons.

ahmedgulabkhan commented:
@alya , can you see if this looks good, thanks

ahmedgulabkhan commented:
Hi @alya, I just need a review from you whenever you get a chance

alya commented:
Thanks, that's looking better, but please add more complete screenshots, including:

- invitations (you need to actually have some in the table!)
- the help center changes
- at least some screenshots for light theme

ahmedgulabkhan commented:
Thanks for the feedback @alya , I have updated the description of the PR to reflect all the above mentioned screenshots, please take a look at your convenience

alya commented:
Great, thank you for the comprehensive screenshots -- looks good to me.

@sahil839 could you please review these changes?

sahil839 commented:
The icons in user list look small and not aligned vertically.
![image](https://github.com/user-attachments/assets/28cccdbd-c82e-46bc-84a0-08d853845b5c)

Also, I am not sure whether we need some attribution for the newly added icons.

ahmedgulabkhan commented:
Hi @sahil839 , are you running my latest code. Below is the screenshot from the latest code that I have and the icons look aligned vertically

![Users-light-after](https://github.com/user-attachments/assets/0f97079f-3f48-4a79-b8c2-988378f697e3)

And also, can you please let me know what you mean by "adding some attribution for the icons", thanks

alya commented:
@ahmedgulabkhan https://zulip.readthedocs.io/en/latest/subsystems/icons.html#adding-a-new-icon

sahil839 commented:
I am seeing this on your PR rebased on main on Chrome.

ahmedgulabkhan commented:
That's strange, the above attached screenshots are what I captured after testing on Chrome

And also I wanted to know how we can add attributions for the below icons in the `docs/THIRDPARTY` file:

user-x
user-cog
user-plus (I see that we already have attributions for this icon)
For reference, we're using the below attribution for the user-plus icon

```
Files: web/shared/icons/user-plus.svg
Copyright: 2013-2023 Cole Bemis (https://feathericons.com)
License: Expat

sahil839 commented:
I tested it again and the icons are still not aligned. Have you rebased your local branch to upstream main?

ahmedgulabkhan commented:
I just rebased my local branch onto upstream main, can you check again and let me know if the icons are still not aligned vertically

sahil839 commented:
The icons are still misaligned for me. I am seeing this at 16px font size and 140 line height.

ahmedgulabkhan commented:
When you say you're seeing the icons still misaligned at 16px font size and 140 line height, do you mean that you've made some changes in css on top of this PR?

sahil839 commented:
No, I am talking about the settings here - 
![image](https://github.com/user-attachments/assets/48e94f64-90b8-472e-b5e0-21f5bf1d9a34)

alya commented:
@ahmedgulabkhan These changes need to be test at 12, 16 and 20 px font size, and all need to look good.

ahmedgulabkhan commented:
Okay, got it. Let me make sure they're aligned for the different font size and line height settings

ahmedgulabkhan commented:
Below are the screenshots for 12px, 16px and 20px font sizes all using 140% as line-height. And all of them are vertically aligned. I'm using Google chrome

### Font Size: 12px, Line Height: 140%
![Screenshot 2025-01-28 at 1 42 31 PM](https://github.com/user-attachments/assets/bd6589e7-82b1-4740-9d7c-7119b68d6976)

### Font Size: 16px, Line Height: 140%
![Screenshot 2025-01-28 at 1 42 53 PM](https://github.com/user-attachments/assets/b7f97977-4bfc-4b72-9750-cdd2accd031d)

### Font Size: 20px, Line Height: 140%
![Screenshot 2025-01-28 at 1 43 11 PM](https://github.com/user-attachments/assets/256f67df-def8-44be-96e9-198220018097)

ahmedgulabkhan commented:
Hi team, any update regarding this?

sahil839 commented:
Maybe sharing on CZO for someone else to test it and check the behavior. Also, the icon for invite link seems misaligned - 
![image](https://github.com/user-attachments/assets/af93d134-5bbb-4248-b079-420a82b66f2e)

ahmedgulabkhan commented:
Sure, I'll share this on CZO and ask someone to test this out

ahmedgulabkhan commented:
Hi @sahil839 , after talking to @Harshbansal8705 , we figured out that the vertical alignment issue was because `vertical-align: middle` not working properly across different operating systems and browsers.

I have made a few changes to the PR and also updated the screenshots in the description of this PR to reflect how the icons look after the change. One thing to notice here is that after making the buttons (which hold the icons) to have `display: inline-flex`, the height of the buttons seem to have reduced slightly in size, let me know if you want me to make changes to get the height back to how it was

sahil839 commented on web/styles/settings.css:
```diff
@@ -1277,9 +1277,39 @@ label.preferences-radio-choice-label {
+.deactivate-bot-button,
+.download_bot_zuliprc,
+.copy_zuliprc,
+.open_bots_subscribed_streams {
```
Were these last three buttons in bots UI also misaligned? If yes, these changes should be in a separate commit.

ahmedgulabkhan replied:
Yes, they were already misaligned. Do you want me to make the changes to the last three buttons in a separate commit of the current PR, or do you want me to raise a new PR for these changes?

sahil839 replied:
Separate commit in this PR would be fine.

ahmedgulabkhan replied:
Added the change to a separate commit and updated the PR description showing before and after screenshots of the misalignment in the icons

sahil839 commented on web/styles/settings.css:
```diff
@@ -1277,9 +1277,39 @@ label.preferences-radio-choice-label {
-    margin-right: 5px;
+
+    &:hover {
+        text-decoration: none;
```
These changes for `text-decoration` also seems unrelated to changing icon. Is this something that fixes something unrelated to changing icons or these are needed due to change in `display` property?

ahmedgulabkhan replied:
So, adding the new icon and using `display: inline-flex` added the underline to the icon as well. So the above change is to remove the underline on hover from the icon. Do you want me to add this in a separate commit?

sahil839 replied:
Ok, I think there is no need for `text-decoration: none` for selectors without `:hover`.

ahmedgulabkhan replied:
I'm sorry but I didn't get what you said

sahil839 replied:
I guess the two places where `text-decoration` is added with `&:hover` are not needed -
```css
&:hover {
        text-decoration: none;
}

.invite-label {
        &:hover {
            text-decoration: underline;
        }
    }
```

ahmedgulabkhan replied:
Removing the `text-decoration` property from the above two places make it look something like this
![Screenshot 2025-02-06 at 10 58 10 PM](https://github.com/user-attachments/assets/a9f1e889-80d9-46f1-ae60-d6bef91b3415)

sahil839 replied:
Ok, I might have done something different while testing. Fine then nothing to change here.

sahil839 commented on web/styles/settings.css:
```diff
@@ -1277,9 +1277,39 @@ label.preferences-radio-choice-label {
+
+.invite-user-link {
+    display: inline-flex;
+    align-items: center;
```
The icon for this still looks misaligned.

ahmedgulabkhan replied:
Is it possible if you can share a screenshot, so I can have reference of how it looks like for you

sahil839 replied:
Feels like text should be couple of pixels below - 
![image](https://github.com/user-attachments/assets/3d8b01e2-b510-4ea7-b11e-7001915ad145)

ahmedgulabkhan replied:
I changed the `line-height` for the `<a>` tag to be `1`, can you check and let me know if that aligns this invite icon vertically

sahil839 replied:
Still does not look perfectly aligned. Maybe we should use grid-based layout like we do in right sidebar.

ahmedgulabkhan replied:
I'll see what I can do. It's just that on my MacOS Chrome and Safari browsers the icon appears aligned, but for a different OS like Linux for example, the icon still is misaligned, which is making it a bit harder for me to test out the behaviour with each change

ahmedgulabkhan replied:
I've made changes to use `display: grid`, let me know if it looks aligned

ahmedgulabkhan replied:
Sure, thanks. Should I reach out to Karl on Zulip chat or just wait for him to give a response in this thread?

sahil839 replied:
Can wait for a couple of days, as it might be easy to help by seeing the code and screenshots.

ahmedgulabkhan replied:
Got it

sahil839 replied:
@karlstolley do you have some time available to look at this? Alignment looks fine at 16px but not for other font sizes.

timabbott replied:
I tested and this seems to no longer be a problem, marking as resolved.

sahil839 reviewed (commented):
Posted some comments. And yes the buttons look small, would be better to set the original height.

ahmedgulabkhan commented:
I have added a min-height property to make the button height to be the same as previously

sahil839 reviewed (commented):
@karlstolley can you have a look at once? I am not sure how to align the icons correctly - both in buttons, which look correct with the current implementation and for invite link.

karlstolley commented:
@sahil839 -- I'm happy to help but I really need some clarity on what it is you'd like me to look at. Is it the tabs? The text is going to look somewhat high in English, simply because there are no descenders (p, q, g, etc.) in those labels...so any adjustment against that is going to cause problems for languages that might include those characters.

ahmedgulabkhan commented:
Hi @karlstolley , thanks for joining in. This PR includes new icons to replace the old ones. I have done the changes at the below places:

### Icon changes:

In Organization Settings:
- Replaced `fa-user-times` with `user-x.svg` in both user and bot management panels
- Replaced `fa-user-plus` with `user-plus.svg` in both user and bot management panels
- Replaced `zulip-icon-user-cog` with `user-manage.svg`, and kept the current icon name
- Replaced `fa-pencil` with `user-manage.svg`(i.e., `zulip-icon-user-cog`) in both the user and bot management panels

In Help Center:
- Replaced `fa-user-times` with `user-x.svg` for deactivating/reactivating user, bot
- Replaced `fa-user-plus` with `user-plus.svg` for deactivating/reactivating user, bot
- Replaced `fa-pencil` with `user-manage.svg`(i.e., `zulip-icon-user-cog`) for managing user, bot

Now, the issue that we were facing is that in the 'Settings' -> 'Organization' -> 'Users' -> 'Invitations' section the new icon (i.e; the user-plus.svg) is not properly aligned vertically for operating systems other than MacOS as you can see in the screenshot below. And we needed your help to check if we're missing something while making the icon to be aligned vertically. @sahil839 , let me know if this messages covers everything or if I need to add more context

I don't think this vertical alignment issue that we're facing is related to the english alphabet letters like 'p', 'q', 'g', 'y' or 'j'. It still looks slightly misaligned if you closely take a look at the screenshot below. The screenshot below is what @sahil839 has been seeing on his machine

![Screenshot 2025-02-26 at 12 08 01 AM](https://github.com/user-attachments/assets/514b9862-8ed8-4d85-a300-fdfa3a392dd2)

ahmedgulabkhan commented:
Sure @sahil839 , I'll squash the commit. I'm waiting for a reply in [this thread](https://chat.zulip.org/#narrow/channel/101-design/topic/deactivate.20group.20icon/near/2105886) from Vlad regarding the attributions that I need to add for the new icons. Once I get the confirmation from him, I'll push that attribution change in this PR, squash the commits based on the guidelines and then update you. Let me know if that sounds good

ahmedgulabkhan commented:
Hi @sahil839 , can you check this PR now and see if it looks good, thanks

sahil839 commented:
Looks good to me. @alya do you want to have a look at this again?

alya commented on help/edit-a-bot.md:
```diff
@@ -10,7 +10,7 @@ active bot in the organization.
 
-1. Click the **pencil** (<i class="fa fa-pencil"></i>) icon on the profile
+1. Click the **manage bot** (<i class="zulip-icon zulip-icon-user-cog"></i>) icon on the profile
    card for the bot you want to edit.
```
```suggestion
   card for the bot you want to manage.
```

alya commented on help/edit-a-bot.md:
```diff
@@ -25,7 +25,7 @@ active bot in the organization.
 
-1. In the **Actions** column, click the **pencil** (<i class="fa fa-pencil"></i>)
+1. In the **Actions** column, click the **manage bot** (<i class="zulip-icon zulip-icon-user-cog"></i>)
    icon for the bot you want to edit.
```
```suggestion
   icon for the bot you want to manage.
```

alya commented:
Looks good to me otherwise. You can tag @laurynmm to review the help center changes when ready.

ahmedgulabkhan commented:
Hi @alya , I have made the changes that you suggested to me - Updated all the places where "edit" was used to "manage" in https://zulip.com/help/edit-a-bot which includes the page title; and also updated all the references of the page titled "Edit a bot" to "Manage a bot"

Tagging @laurynmm to review the help center changes

alya commented:
@laurynmm This is a nice-to-have for the release.

laurynmm commented:
**Updates**:
- Added a prep commit to show bot in the tooltips for reactivating or deactivating bots in the organization settings overlay.
- Added a prep commit to show a tooltip when a user cannot be deactivated in the organization setting overlay.
- Split out the icon changes so that each of the three icon changes are in a separate commit.
- Split out the help center changes for changing "Edit a bot" to "Manage a bot", and did the redirect for that article title change.
- Split out the icon and CSS changes to the invite users link in the organization settings for invitations.

**Note**: I didn't manually retest on different font settings and line spacing settings.

---

**Screenshots**:

### Org settings - users tab:
| Before | After |
| --- | --- |
| ![Screenshot from 2025-03-15 21-52-49](https://github.com/user-attachments/assets/29ae1fe1-b681-4cca-9038-29e56674c1e1) | ![Screenshot from 2025-03-15 21-55-04](https://github.com/user-attachments/assets/810e59a5-f451-41fc-ae36-89a2523b96d7) |
| ![Screenshot from 2025-03-15 21-52-33](https://github.com/user-attachments/assets/f5ec5148-9dbc-4afe-90ee-1ac41256c22d) | ![Screenshot from 2025-03-15 21-54-53](https://github.com/user-attachments/assets/f8ab6c58-d78a-435d-ab66-9479a8e3bccc) |
| ![Screenshot from 2025-03-15 21-52-25](https://github.com/user-attachments/assets/84da7da1-64f2-4df0-90d3-8498f7e99255) | ![Screenshot from 2025-03-15 21-54-46](https://github.com/user-attachments/assets/1c125806-db98-4207-9de2-15a3e2b9eb30) |
| ![Screenshot from 2025-03-15 21-52-22](https://github.com/user-attachments/assets/497a9f6f-8e55-4a9f-afe5-b5ed348b779a) | ![Screenshot from 2025-03-15 21-54-42](https://github.com/user-attachments/assets/cf0cf17a-0f47-461d-bf31-ad62a53c9763) |
| ![Screenshot from 2025-03-15 21-52-15](https://github.com/user-attachments/assets/afc90574-243d-4e80-9a2a-af6f62ca442e) | ![Screenshot from 2025-03-15 21-53-48](https://github.com/user-attachments/assets/583bf9ab-435e-4687-a1e5-0433f900b8f8) |

### Org settings - bots tab:
| Before | After |
| --- | --- |
| ![Screenshot from 2025-03-15 21-51-26](https://github.com/user-attachments/assets/2a4814ed-b680-4925-978f-856a3a337a8c) | ![Screenshot from 2025-03-15 21-48-17](https://github.com/user-attachments/assets/22e172c4-6177-4dc7-bd9f-8a44d74f3797) |
| ![Screenshot from 2025-03-15 21-50-36](https://github.com/user-attachments/assets/c2de609e-09b8-4eb1-b8e6-e77fdf60dd4c) | ![Screenshot from 2025-03-15 21-47-35](https://github.com/user-attachments/assets/c89f7ac2-7e52-450a-8172-890a313614cb) |
| ![Screenshot from 2025-03-15 21-50-23](https://github.com/user-attachments/assets/44f7943c-f947-4990-9324-938e627b19be) | ![Screenshot from 2025-03-15 21-48-30](https://github.com/user-attachments/assets/3a9095f1-a9d9-4458-8dde-007c85640354) |
| ![Screenshot from 2025-03-15 21-50-12](https://github.com/user-attachments/assets/88a9312f-4809-4183-87a0-8be0d668123b) | ![Screenshot from 2025-03-15 21-47-46](https://github.com/user-attachments/assets/8a2be8c2-3d30-4167-b940-6069eaec7797) |
| ![Screenshot from 2025-03-15 21-50-07](https://github.com/user-attachments/assets/0477b437-f576-4a5f-9e28-76697cdd62d9) | ![Screenshot from 2025-03-15 21-48-11](https://github.com/user-attachments/assets/346272a8-436f-4d5a-b981-bca002df3abb) |

### Org settings - invitations panel:
| Before | After |
| --- | --- |
| ![Screenshot from 2025-03-15 21-46-26](https://github.com/user-attachments/assets/b856ffe5-e1a4-4c33-be7c-31cd4d8bd1cc) | ![Screenshot from 2025-03-15 21-46-51](https://github.com/user-attachments/assets/2ce6c759-19a2-4d4a-95dd-301be73affae) |

### Personal settings - bots tab:
| Before | After |
| --- | --- |
| ![Screenshot from 2025-03-15 21-45-50](https://github.com/user-attachments/assets/660301d3-15f0-496b-ac37-addd2ad23e79) | ![Screenshot from 2025-03-15 21-45-22](https://github.com/user-attachments/assets/8a192dcf-67b6-437b-812a-09d7c5268a04) |
| ![Screenshot from 2025-03-15 21-45-46](https://github.com/user-attachments/assets/c5507ec2-b146-480e-b689-c56ddb9fafea) | ![Screenshot from 2025-03-15 21-45-17](https://github.com/user-attachments/assets/9064331f-f2d4-4e06-b8e0-3f763826a05a) |
| ![Screenshot from 2025-03-15 21-45-41](https://github.com/user-attachments/assets/1ff0c092-5763-4ccb-96d3-74ba65c23088) | ![Screenshot from 2025-03-15 21-45-13](https://github.com/user-attachments/assets/8e5e3e3a-c2af-47ac-b785-0c367787a451) |

### Help center changes:
| Location | Updates |
| --- | --- |
| [Edit a bot](https://zulip.com/help/edit-a-bot) | ![Screenshot from 2025-03-15 21-56-18](https://github.com/user-attachments/assets/8c480747-2797-4b7f-aa6b-70de6ac9e79d) |
| [Generate URL for integration](https://zulip.com/help/generate-integration-url) via org settings | ![Screenshot from 2025-03-15 21-59-59](https://github.com/user-attachments/assets/e24713fe-78a0-44e3-a598-5b8bef2d1597) |
| [Generate URL for integration](https://zulip.com/help/generate-integration-url) via personal settings | ![Screenshot from 2025-03-15 21-59-51](https://github.com/user-attachments/assets/a09d12ec-48f2-4084-9076-87cfefa0d8bb) |
| [Deactivate/Reactivate bot](https://zulip.com/help/deactivate-or-reactivate-a-bot) via personal settings | ![Screenshot from 2025-03-15 21-57-24](https://github.com/user-attachments/assets/c065bb47-1612-4118-b576-7b7d25f9070e) |
| [Deactivate/Reactivate bot](https://zulip.com/help/deactivate-or-reactivate-a-bot) via org settings | ![Screenshot from 2025-03-15 21-57-35](https://github.com/user-attachments/assets/5862931b-7399-48b7-88ec-8e188e34f08b) |
| [Deactivate/Reactivate user](https://zulip.com/help/deactivate-or-reactivate-a-user#deactivate-a-user) | ![Screenshot from 2025-03-15 21-58-58](https://github.com/user-attachments/assets/7e016670-64cd-4ec5-aa53-c887dd7ce34a) |
| [Manage a user ](https://zulip.com/help/manage-a-user)| ![Screenshot from 2025-03-15 21-58-18](https://github.com/user-attachments/assets/dcd38c70-8704-43dc-8c74-f694584d4cab) |

alya commented on help/deactivate-or-reactivate-a-user.md:
```diff
@@ -56,8 +56,8 @@ When you deactivate a user:
 
-1. In the **Actions** column, click the **deactivate** (<i class="fa
-   fa-user-times"></i>) icon for the user you want to deactivate.
+1. In the **Actions** column, click the **deactivate** (<i class="zulip-icon
```
```suggestion
1. In the **Actions** column, click the **deactivate user** (<i class="zulip-icon
```

timabbott replied:
Fixed.

alya commented:
Looks good to me other than the small tweak above. I didn't try to verify completeness of changes.

timabbott commented on web/styles/settings.css:
```diff
@@ -1324,6 +1324,12 @@ label.preferences-radio-choice-label {
+.reactivate-user-button {
+    display: inline-flex;
+    align-items: center;
+    min-height: 35px;
```
Generally, new sizes should always be done in `em`.

timabbott replied:
But I think we can adjust that in a follow-up; we will eventually probably bulk-convert these.

timabbott commented:
Looks great, merged, with the one tweak noted above.
